myhpi/core/templates/core/information_page.html

```diff
@@ -15,30 +15,24 @@ <h1 class="page-title">
     </div>
     <div class="col-lg-3">
         <div class="side-panel-container">
+            {% if request.is_preview %}
+                <aside class="side-panel border-accent toc-container d-print-none">
+                    <h1 class="side-panel-warning">{% translate "⚠️ Preview ⚠️" %}</h1>
```

Comment:
why is this not part of core/sidebar.html?

Reply:
This is shown above all other entries in the sidebar. However, `core/sidebar.html` is copied below some other entries, depending on page type.

Reply:
but is it necessary to have this on top? I just don't like the duplication between the two templates

Reply:
It isn't. However, the label wouldn't be displayed consistently on top then. I guess i could deduplicate it by extracting the code into yet another html file. Would that be better?

Reply:
No I think that only makes it more complicated. Either leave it this way or put it in the sidebar.html and we accept that it doesn't always show on top. I vote for the latter

Reply:
Done.